content/docs/optimizing-performance.md

```diff
@@ -180,6 +180,34 @@ Note that **the numbers are relative so components will render faster in product
 
 Currently Chrome, Edge, and IE are the only browsers supporting this feature, but we use the standard [User Timing API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API) so we expect more browsers to add support for it.
 
+## Visualize Component Updates with React DevTools' Highlight Updates Feature
+
+<img src="https://media.giphy.com/media/xUOxfgXNvzy37XlueY/giphy.gif" style="max-width:100%; margin-top:20px;" alt="React DevTools Highlight Updates example" />
```

Comment:
I think we might want to re-record a gif without recommendation to use Redux. Since in this context it appears as if it's *us* recommending it :-)

Reply:
Sure thing.